lib/decorators/component-scan.decorator.ts

```diff
@@ -27,22 +30,21 @@ function getForRootPath(pathParameter?: string | string[]): string[] {
   return Array.isArray(pathParameter) ? pathParameter.map(toForRootPath) : [pathParameter].map(toForRootPath);
```

Comment:
I think we have to avoid 3-term operator usage.
It'll threat our code with readability and side effects.
How about with condition blocks like below ?
```typescript
if (Array.isArray(pathParameter)) {
  return pathParameter.map(toForRootPath)
}
return [pathParameter].map(toForRootPath)
```

Reply:
Thank you. I made a slight change based on your review. 
I modified the line 
`if (Array.isArray(pathParameter)) return pathParameter.map(toForRootPath);` to match the pattern of 
`if (!pathParameter) return [getFolderPath()];`

so the final code is now
```
function getForRootPath(pathParameter?: string | string[]): string[] {
  if (!pathParameter) return [getFolderPath()];
  if (Array.isArray(pathParameter)) return pathParameter.map(toForRootPath);
  return [pathParameter].map(toForRootPath);
}
```